src/test/java/LoginTests.java

```diff
@@ -65,5 +69,23 @@ public static void updateProfileNameTest () throws InterruptedException {
 
     }
 
+    public class Homework16 extends BaseTest {
```

Comment:
Please create a new class and write the code

For example

![image](https://user-images.githubusercontent.com/104761482/210740035-4c831486-9068-4f79-9463-32e94bd4bb0d.png)